### PR TITLE
[13.0][MIG] sale_order_line_date

### PR DESCRIPTION
**Pull request by NuriaXifre**
Migration of the module sale_order_line_date to v13.
cc ~ @ForgeFlow

### CODE REVIEW

File: sale_order_line_date/migrations/12.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 ForgeFlow (https://www.forgeflow.com)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openupgradelib import openupgrade
```
Comment (AdriaGForgeFlow) [inline sale_order_line_date/migrations/12.0.1.0.0/pre-migration.py]: This is no longer needed, you can remove it.

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Code review :+1: 

Just a comment below:

**Review by AdriaGForgeFlow — APPROVED** [review]

**Comment by rousseldenis** [comment]
@NuriaMForgeFlow Could you check isort configuration as travis failed?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on runbot. Works as expected.

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_order_line_date/models/sale_order_line.py [hunk 2]
```diff
@@ -12,9 +12,8 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    commitment_date = fields.Datetime(oldname="requested_date")
+    commitment_date = fields.Datetime()
```
Comment (MiquelRForgeFlow) [inline sale_order_line_date/models/sale_order_line.py]:
```suggestion
    commitment_date = fields.Datetime('Delivery Date')
```
Like in sale orders.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@rousseldenis ready to merge I guess ;)

**Pull request merged**